frontend/src/pages/about/Donate.tsx

```diff
@@ -41,130 +32,130 @@ const useStyles = makeStyles((theme) => ({
 
 const DonatePage = () => {
   const { t } = useTranslation();
+
   const classes = useStyles();
   return (
     <>
-      <ContentHeader title="About > Donate" />
-      <div className={classes.root}>
-        <div className={classes.content}>
-          <Typography paragraph>
-            <Trans t={t} i18nKey="about.donateWeAreASmallTeam">
-              Because we are a small team of mostly volunteers, the development
-              of Tournesol is slower than we would like it to be. If you can,
-              please consider helping us, through coding or through donations.
-              Check-out our{' '}
-              <a
-                href={githubTournesolUrl}
-                target="_blank"
-                rel="noreferrer"
-                className={classes.link}
-              >
-                open source code
-              </a>
-              , or join our{' '}
-              <a
-                href={discordTournesolInviteUrl}
-                target="_blank"
-                rel="noreferrer"
-                className={classes.link}
-              >
-                Discord
-              </a>
-              .
-            </Trans>
-          </Typography>
+      <ContentHeader title={t('about.donate')} />
+      <FundingSection linkToSupportPage={false} />
```

Comment:
We should move `<FundingSection>` inside `<ContentBox maxWidth="lg">` to include it in the page container. It will automatically benefits from the margins and padding defined by `ContentBox`.